src/install_rules.ml

```diff
@@ -167,12 +167,24 @@ module Gen(P : Params) = struct
         ~has_native:(Option.is_some ctx.ocamlopt)
     in
     let if_ cond l = if cond then l else [] in
+    let installable_modules =
+      Dir_contents.modules_of_library dir_contents
+        ~name:(Library.best_name lib)
+      |> Lib_modules.installable_modules
+    in
+    let sources =
+      List.concat_map installable_modules ~f:(fun m ->
+        List.map (Module.sources m) ~f:(fun source ->
+          let dst =
```

Comment:
Slight shorter, and does something similar: `Path.basename source |> String.drop_suffix ~suffix:"-gen"`

Reply:
I pushed this change

Reply:
Thanks